products/dns/inspec.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+manifest: !ruby/object:Provider::Inspec::Manifest
+  summary: 'InSpec resources for verifying GCP infrastructure'
+  description: |
+    InSpec resources for verifying GCP infrastructure
+overrides: !ruby/object:Overrides::ResourceOverrides
+  ManagedZone: !ruby/object:Overrides::Inspec::ResourceOverride
```

Comment:
Didn't you do managed zones already? Or did I imagine that?

Reply:
Had a PR for it, removed it because a handwritten resource exists in inspec-gcp and I don't want to overwrite anything yet.

Reply:
Awesome. Can you put a comment for those things? It would be nice to know why certain excludes exist. (Is it because it's unnecessary in Inspec, handwritten, just haven't gotten to it yet?)